components/src/BulkOrder/bulkorder.main.tsx

```diff
@@ -200,6 +209,43 @@ export class BulkOrder extends React.Component<BulkOrderProps, BulkOrderState> {
     }
   }
 
+  handleBarcodeClick() {
+    this.setState({ isBarcodeScannerOpen: true });
+  }
+
+  handleBarcodeModalClose() {
+    this.setState({ isBarcodeScannerOpen: false });
+  }
+
+  handleBarcodeScanned(barcode) {
+    login().then(() => {
+      searchLookup(barcode)
+        .then((res) => {
+          const foundItem = res._element[0];
```

Comment:
If no item is found, we should display an error message. You can re-use the `no-results-found` message for this 😄